packages/design-system/src/components/Table/Table.jsx

```diff
@@ -0,0 +1,145 @@
+import React, { useEffect, useRef, useState } from 'react';
+import Alert from '../Alert/Alert';
+import PropTypes from 'prop-types';
+import TableCaption from './TableCaption';
+import classNames from 'classnames';
+
+function debounce(fn, ms) {
+  let timer;
+  return () => {
+    clearTimeout(timer);
+    timer = setTimeout(() => {
+      timer = null;
+      fn.apply(this, arguments);
+    }, ms);
+  };
+}
+
+export const Table = ({
+  className,
+  responsiveTable,
+  striped,
+  scrollable,
+  scrollableAlert,
+  children,
+  ...attributeOptions
+}) => {
+  const container = useRef(null);
+  // The captionID is calculated and stored as init value of a ref.
+  // This ensures that the ID remains constant for all renders.
+  const captionID = useRef('caption-' + Math.random().toString(36).substr(2, 9));
+  const [tabIndex, setTabIndex] = useState(null);
```

Comment:
Instead of having `tabIndex` inside the state, what do you think about using a boolean named `scrollable` or `isTableScrollable`. This is more descriptive than tab index, and you can use it directly instead of including logic like this: `const isActive = tabIndex === '0';`

Reply:
Good point.